macros/schema_tests/multi-column/expect_select_column_values_to_be_unique_within_record.sql

```diff
@@ -40,7 +40,7 @@
 with column_values as (
 
     select
-        row_number() over(order by 1) as row_index,
+        row_number() over(order by {{ columns|join(', ') }}) as row_index,
```

Comment:
Was this failing? Is this required for Trino support? I'd have to look at this separately to step through, it's been a while, but if this is a bug, I'd rather roll this into a separate PR.

Reply:
In Trino, ordering by column's position number inside ranking functions doesn't work. `row_number() over(order by 1)` results in a random order. [This test](https://github.com/calogica/dbt-expectations/blob/0.10.1/integration_tests/models/schema_tests/schema.yml#L474-L476) fails because of that.

However, IMHO it is a bug anyway. 

Mentioned test passes in other dbt adapters, because in [current test model](https://github.com/calogica/dbt-expectations/blob/0.10.1/integration_tests/models/schema_tests/data_test.sql) the first column `idx` contains subsequent id numbers. But, for example, we could have same value for all rows in the first column, and we will receive a random order.

IMHO ordering by all columns that are expected to have unique values makes more sense, than ordering by the first arbitrary column.

WDYT? I can create a separate PR for this change, if you agree with the above.